src/sqlfluff/cli/commands.py

```diff
@@ -175,14 +175,14 @@ def core_options(f: Callable) -> Callable:
         f = click.option(
             "--dialect",
             default=None,
-            help="The dialect of SQL to lint (default=ansi)",
+            help="The dialect of SQL to lint",
```

Comment:
I wonder if we should make this mandatory?

```suggestion
            help="The dialect of SQL to lint (mandatory)",
```

Reply:
You can also specify the dialect in `.sqlfluff`, so I wonder if this would be confusing.

Reply:
Fair point. Let's leave it as is.